code/game/objects/items/food/bread.dm

```diff
@@ -325,6 +325,18 @@
 
 /obj/item/food/deepfryholder/proc/On_Consume(eater, feeder)
 	if(contents)
+		if(isliving(eater))
```

Comment:
early return both of these (second check should `QDEL_LIST(contents)` before returning)